lldb/test/API/python_api/sbsavecoreoptions/TestSBSaveCoreOptions.py

```diff
@@ -79,3 +81,26 @@ def test_adding_thread_different_process(self):
         self.assertTrue(error.Fail())
         error = options.AddThread(thread)
         self.assertTrue(error.Success())
+
+    def test_removing_and_adding_insertion_order(self):
+        """Test insertion order is maintained when removing and adding threads."""
```

Comment:
The comment for `SBSaveCoreOptions::GetThreadsToSave` says you get an unsorted collection back. Which one is the intended behavior?

Reply:
Hey Alex, good catch. This shouldn't test the ordering, it did before when used our own internal collection but after Greg and I had some back and forth now we're returning a thread collection.